dotcom-rendering/src/lib/frontsBannerAbTestAdPositions.ts

```diff
@@ -58,6 +58,15 @@ export const sectionFrontsBannerAdCollections: FrontsBannerAdCollections = {
 		'Video',
 		'Other sports',
 	],
+	'us/sport': [
+		// multiline array
+		'Around the world',
+		'Video',
+	],
+	'au/sport': [
+		'Woman’s World Cup 2023', // Apostrophe is intentionally U+2019 "’" instead of the more common ASCII character U+0060 "`",
```

Comment:
Is the womens world cup 2023 section going to stick around for the duration of the test?

Reply:
Who knows, it's been removed from the network fronts but not this one, should they be updated you recon?

Reply:
Check with Alex. It might be planned to stay there for another month. If it's not clear, then it might be safer to avoid it.

Reply:
I think we will have to keep an eye on them during the test anyways and move the banners, as I've seen changes to a couple even while I've worked on this PR.